.gitlab-ci.yml

```diff
@@ -144,7 +144,8 @@ test:
     - time cargo fetch
     - time cargo fetch --manifest-path=`cargo metadata --format-version=1 | jq --compact-output --raw-output  ".packages[] | select(.name == \"polkadot-test-runtime\").manifest_path"`
     - time cargo fetch --manifest-path=`cargo metadata --format-version=1 | jq --compact-output --raw-output  ".packages[] | select(.name == \"polkadot-runtime\").manifest_path"`
-    - CARGO_NET_OFFLINE=true time cargo test --verbose --workspace
+    - time cargo fetch --manifest-path=`cargo metadata --format-version=1 | jq --compact-output --raw-output  ".packages[] | select(.name == \"kusama-runtime\").manifest_path"`
```

Comment:
Originally this cargo fetch has been added because we had problems with concurrent runtime builds (we build 6 now :/) and `cargo metdata` was called during build, which doesn't work well with concurrent builds. I've added `SKIP_POLKADOT_RUNTIME_WASM_BUILD` and etc because we don't need these runtimes in `cargo test`.